test/spec/modules/vidazooBidAdapter_spec.js

```diff
@@ -0,0 +1,188 @@
+    });
+
+    it('should require pId', () => {
+      const isValid = adapter.isBidRequestValid({
+        params: {
+          cId: 'cid'
+        }
+      });
+      expect(isValid).to.be.false;
+    });
+
+    it('should validate correctly', () => {
+      const isValid = adapter.isBidRequestValid({
+        params: {
+          cId: 'cid',
+          pId: 'pid'
+        }
+      });
+      expect(isValid).to.be.true;
+    });
+  });
+
+  describe('build requests', () => {
+    let sandbox;
+    before(() => {
+      sandbox = sinon.sandbox.create();
+      sandbox.stub(utils, 'getTopWindowUrl', () => 'http://www.greatsite.com');
+      sandbox.stub(Date, 'now', () => 1000);
+    });
+
+    it('should build request for each size', () => {
+      const requests = adapter.buildRequests([BID]);
+      expect(requests).to.have.length(2);
+      expect(requests[0]).to.deep.equal({
+        method: 'GET',
+        url: `${URL}/prebid/59db6b3b4ffaa70004f45cdc`,
+        data: {
+          width: 300,
+          height: 250,
+          url: 'http://www.greatsite.com',
+          cb: 1000,
+          bidFloor: 0.1,
+          bidId: '2d52001cabd527',
+          publisherId: '59ac17c192832d0011283fe3'
+        }
+      });
+      expect(requests[1]).to.deep.equal({
+        method: 'GET',
+        url: `${URL}/prebid/59db6b3b4ffaa70004f45cdc`,
+        data: {
+          width: 300,
+          height: 600,
+          url: 'http://www.greatsite.com',
+          cb: 1000,
+          bidFloor: 0.1,
+          bidId: '2d52001cabd527',
+          publisherId: '59ac17c192832d0011283fe3'
+        }
+      });
+    });
+
+    after(() => {
+      sandbox.restore();
+    });
+  });
+
+  describe('interpret response', () => {
+    it('should return empty array when there is no response', () => {
+      const responses = adapter.interpretResponse(null);
+      expect(responses).to.be.empty;
+    });
+
+    it('should return empty array when there is no ad', () => {
+      const responses = adapter.interpretResponse({price: 1, ad: ''});
+      expect(responses).to.be.empty;
+    });
+
+    it('should return empty array when there is no price', () => {
+      const responses = adapter.interpretResponse({price: null, ad: 'great ad'});
+      expect(responses).to.be.empty;
+    });
+
+    it('should return an array of interpreted responses', () => {
+      const responses = adapter.interpretResponse(SERVER_RESPONSE, REQUEST);
+      expect(responses).to.have.length(1);
+      expect(responses[0]).to.deep.equal({
+        requestId: '2d52001cabd527',
+        bidderCode: 'vidazoo',
+        cpm: 0.8,
+        width: 300,
+        height: 250,
+        creativeId: '12610997325162499419',
+        currency: 'USD',
+        netRevenue: true,
+        ttl: 30,
+        ad: '<iframe>console.log("hello world")</iframe>'
+      });
+    });
+
+    it('should take default TTL', () => {
+      const serverResponse = utils.cloneJson(SERVER_RESPONSE);
```

Comment:
In #1910 `utils.cloneJson` was renamed to `utils.deepClone`, updating this should get the test passing again